diff --git a/dist/shared/better-auth.cM0Fc3LZ.mjs b/dist/shared/better-auth.cM0Fc3LZ.mjs
index 80e7eecdf86b6ddab57c74ed51156a635fed63fa..2deb583132b299d0a3e877b43fd6f49d005599fe 100644
--- a/dist/shared/better-auth.cM0Fc3LZ.mjs
+++ b/dist/shared/better-auth.cM0Fc3LZ.mjs
@@ -1787,7 +1787,10 @@ const signInEmail = createAuthEndpoint(
         session,
         user: user.user
       },
-      ctx.body.rememberMe === false
+      ctx.body.rememberMe === false,
+      {
+        domain: ".asyncstatus.com",
+      }
     );
     return ctx.json({
       redirect: !!ctx.body.callbackURL,
